xtask/src/run/rust_crate.rs

```diff
@@ -21,6 +21,12 @@ pub(crate) struct CrateArg {
     #[clap(long, requires = "crate_dir", default_value = "false")]
     pub(crate) release: bool,
 
+    /// Use a specific build profile
+    ///
+    /// This overrides the release flag if both are specified.
+    #[clap(long, requires = "crate_dir")]
```

Comment:
This can be a bit fiddly sometimes, but I think adding `conflicts_with_all` to either this attribute or `release` should make `--profile` and `--release` mutually exclusive.

```suggestion
    #[clap(long, requires = "crate_dir", conflicts_with_all = ["release"])]
```

Reply:
Oh, nice! I wasn't aware of this feature. 🤯